src/main/java/org/cascadebot/cascadebot/events/CommandListener.java

```diff
@@ -81,10 +82,17 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
         String trigger;
         String[] args;
 
+        boolean messagesBot = false;
+        for (Member member : event.getMessage().getMentionedMembers()) {
+            if (member.getIdLong() == event.getGuild().getSelfMember().getIdLong()) {
+                messagesBot = true;
+            }
+        }
+
```

Comment:
This won't work as intended as you could mention the bot at any point in the message.

We need to do what we did but just use maybe regex for it to match the extra character.